services/userServices.js

```diff
@@ -0,0 +1,29 @@
+const jwt = require('jsonwebtoken');
+require('dotenv').config();
+const userDao = require('../models/userDao');
+const { axiosUser } = require('../util/axios');
+
+const loginService = async (kakaoToken) => {
+  try {
+    const user = await axiosUser(kakaoToken);
+    const id = user.data.id;
+    const image = user.data.properties.profile_image || null;
+    const name = user.data.properties.nickname || null;
+    const email = user.data.kakao_account.email;
+    const isExist = await userDao.getUserById(id);
```

Comment:
해당 부분 변수 수정이 안된 것 같아요!
더 명확한 변수를 사용할 수 있도록 수정해주세요

```suggestion
    const user = await userDao.getUserById(id);
```